rca/enquire_to_study/templates/enquire_to_study/index.html

```diff
@@ -15,92 +15,53 @@
+                                {% if view.get_page_subtitle %} <span class="w-header__subtitle">{{ view.get_page_subtitle }}</span> {% endif %}
+                                {% include 'modeladmin/includes/result_count.html' %}
+                            </h1>
+                        {% endblock %}
                     </div>
                     {% block search %}{% search_form %}{% endblock %}
                 </div>
                 {% block header_extra %}
-                    <div class="right">
-                        {% if user_can_create %}
+                    <div class="right header-right">
+                        {% if view.list_export or user_can_create %}
                             <div class="actionbutton">
-                                {% include 'modeladmin/includes/button.html' with button=view.button_helper.add_button %}
-                            </div>
-                        {% endif %}
-                        {% if view.list_export %}
-                            <div class="dropdown dropdown-button match-width col">
-                                <a href="?export=xlsx&{{ request.GET.urlencode }}" class="button bicolor button--icon">{% icon name="download" wrapped=1 %}{% trans 'Download XLSX' %}</a>
-                                <div class="dropdown-toggle">{% icon name="arrow-down" %}</div>
-                                <ul>
-                                    <li><a  class="button bicolor button--icon" href="?export=csv&{{ request.GET.urlencode }}">{% icon name="download" wrapped=1 %}{% trans 'Download CSV' %}</a></li>
-                                </ul>
+                                {% if user_can_create %}
+                                    {% include 'modeladmin/includes/button.html' with button=view.button_helper.add_button %}
+                                {% endif %}
+                                {% if view.list_export %}
+                                    <div class="dropdown dropdown-button match-width col">
+                                        <a href="?export=xlsx&{{ request.GET.urlencode }}" class="button bicolor button--icon">{% icon name="download" wrapped=1 %}{% trans 'Download XLSX' %}</a>
+                                        <div class="dropdown-toggle">{% icon name="arrow-down" %}</div>
+                                        <ul>
+                                            <li><a  class="button bicolor button--icon" href="?export=csv&{{ request.GET.urlencode }}">{% icon name="download" wrapped=1 %}{% trans 'Download CSV' %}</a></li>
+                                        </ul>
+                                    </div>
+                                {% endif %}
+                                <a  class="button button--icon no" href="{% url 'enquiretostudy_delete' %}">
+                                    {% icon name="bin" %}
+                                    {% trans 'Delete submissions' %}
+                                </a>
```

Comment:
This header block, I got from the newest from the modeladmin index header and added the delete button.

Reply:
Same great work here!